k8s-plugin/src/test/kotlin/com/amazon/spinnaker/keel/tests/HelmResourceHandlerTest.kt

```diff
@@ -174,147 +342,213 @@ internal class HelmResourceHandlerTest : JUnit5Minutests {
                 clearAllMocks()
             }
 
-            context("with invalid resource spec") {
-                var spec = yamlMapper.readValue(wrongYaml, HelmResourceSpec::class.java)
-                var resource = resource(
-                    kind = HELM_RESOURCE_SPEC_V1.kind,
-                    spec = spec
```

Comment:
not testing invalid resource spec anymore?

Reply:
Added tests along with verification function